### PR TITLE
feature(common):add menu

### PR DESCRIPTION
**Pull request by choquevillca**
menu design

### CODE REVIEW

File: src/components/Sidebar/Menu.scss [hunk 1]
```diff
@@ -0,0 +1,186 @@
+@mixin sidebar {
```
Comment (Ditmar) [inline src/components/Sidebar/Menu.scss:1]: don't use sass for this we use material with makeStyles please change this

**Review by Ditmar — CHANGES_REQUESTED** [review]
Remove sass from the pr, and use material ui with the definitions that I had taught in class.

File: src/components/Sidebar/Menu.scss (hunk 1 above)
Comment (Ditmar) [reply src/components/Sidebar/Menu.scss:1]: please delete sass, we are using material

**Review by Ditmar — CHANGES_REQUESTED** [review]
you must to use material instead sass

File: src/components/Sidebar/Menu.scss (hunk 1 above)
Comment (choquevillca) [reply src/components/Sidebar/Menu.scss]: resolve

**Comment by choquevillca** [comment]
please review revised changes

File: src/components/Sidebar/menu.styles.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import {Button} from '@material-ui/core'
+import {makeStyles, withStyles} from '@material-ui/styles';
+export const BootstrapButton = withStyles({
+    root: {
```
Comment (Ditmar) [inline src/components/Sidebar/menu.styles.js:45]: the function of material ui is makeStyles that function join the themplate with your components

File: src/components/Sidebar/useMenuComponent.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    return setSelectedItem(newItem);
+  };
+
+  return {
+    isOpen,
+    setIsOpen,
+    isActive,
+    onMenuItemClicked,
+  };
```
Comment (Ditmar) [inline src/components/Sidebar/useMenuComponent.js:32]: nice hook

File: src/components/Sidebar/MenuComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+import IconContacts from "./iconos/icon-contacts";
+import IconAgents from "./iconos/icon-agents";
+import IconArticles from "./iconos/icon-articles";
+import IconSettings from "./iconos/icon-settings";
+import { useStyles } from "./menu.styles";
+import { slide as Menu } from "react-burger-menu";
+import useMenuComponent from "./useMenuComponent";
+
+const styles = {
+  bmBurgerButton: {
```
Comment (Ditmar) [inline src/components/Sidebar/MenuComponent.jsx]: The styles should be in a different location.

**Review by Ditmar — CHANGES_REQUESTED** [review]
The pr is good but the styles is not according to the standard, please hurry up and fix it

**Review by Ditmar — APPROVED** [review]

**Comment by choquevillca** [comment]
resolved conflicts, check it please

**Pull request merged**